plugins/tailor.cpp

```diff
@@ -666,9 +668,10 @@ DFhackCExport command_result plugin_load_site_data (color_ostream &out) {
         DEBUG(control,out).print("no config found in this save; initializing\n");
         config = World::AddPersistentSiteData(CONFIG_KEY);
         config.set_bool(CONFIG_IS_ENABLED, is_enabled);
+        config.set_bool(CONFIG_CONFISCATE, true);
     }
-
     is_enabled = config.get_bool(CONFIG_IS_ENABLED);
+    confiscate = config.get_bool(CONFIG_CONFISCATE);
```

Comment:
this will cause all existing saves to disable confiscation. it's much safer to have your boolean state default to false. Maybe rename it `CONFIG_NO_CONFISCATE`